bioframe/core/stringops.py

```diff
@@ -11,7 +11,7 @@
 ### functions for manipulating UCSC strings ###
 
 
-def to_UCSC_string(triplet):
+def to_ucsc_string(triplet):
```

Comment:
for stringops, I think we should add `is_complete_ucsc_string` and `to_ucsc_string` to _all_